### PR TITLE
basic speedometer in ui

### PR DESCRIPTION
**Pull request by laurengarcia**
this works well, but the archive seems to be emitting no 'upload' events. will ping mafintosh about it. 
download works well though, and upload logic is identical here so whenever 'upload' starts working this will just work, too.

### CODE REVIEW

**Comment by okdistribute** [comment]
great!!

File: app.js [hunk 1]
```diff
@@ -88,3 +91,34 @@ function installDropHandler (archive) {
...
+  var $els = {
+    speed: document.getElementById('speed'),
+    upload: document.getElementById('upload-speed'),
+    download: document.getElementById('download-speed')
+  }
+  var timer;
+  function update (direction, data) {
+    if (!data.length) return
+    var bytesPerSecond = speed(data.length)
+    if (bytesPerSecond && $els[direction]) {
```
Comment (okdistribute) [inline app.js:106]: slick

File: app.js [hunk 2]
```diff
@@ -88,3 +91,34 @@ function installDropHandler (archive) {
...
+  var speed = speedometer(1)
+  var $els = {
+    speed: document.getElementById('speed'),
+    upload: document.getElementById('upload-speed'),
+    download: document.getElementById('download-speed')
+  }
+  var timer;
+  function update (direction, data) {
+    if (!data.length) return
+    var bytesPerSecond = speed(data.length)
```
Comment (mafintosh) [inline app.js:105]: is this using the same speedometer for both upload / download? you need one for each :)

Comment (okdistribute) [inline app.js:105]: good point

**Pull request merged**